tomography_preprocessing/tilt_series_alignment/aretomo.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+from typing import Optional
+
+import typer
+from lil_aretomo.aretomo import run_aretomo_alignment
+
+from ._cli import cli
+from .. import utils
+
+ARETOMO_ALIGNMENT_COMMAND_NAME = 'AreTomo'
+
+@cli.command(name=ARETOMO_ALIGNMENT_COMMAND_NAME)
+def aretomo_function(
+        tilt_series_star_file: Path = typer.Option(...),
+        output_directory: Path = typer.Option(...),
+        exe: Path = typer.Option(...),	
+        local_align: Optional[bool] = typer.Option(False),
+        target_pixel_size: Optional[float] = typer.Option(10),
+        patch_in_x: Optional[float] = typer.Option(5),
+        patch_in_y: Optional[float] = typer.Option(4),
```

Comment:
refactor to `patch_size_px: Tuple[int, int]`

Reply:
Read the API and this is wrong... my bad - I would still prefer something along the lines of num_patches_xy: Tuple[int, int]

Reply:
Fixed in the latest version of LilAreTomo, and will be integrated  in commit 'patch_tuple' here